recipes/nng/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+mkdir build
+
+if [[ "$target_platform" == osx-arm64 ]]; then
+	cmake -B build -S . \
+		${CMAKE_ARGS} \
+                -G Ninja \
+		-DCMAKE_INSTALL_PREFIX=$PREFIX \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_VERBOSE_MAKEFILE=ON \
+		-DENABLE_TESTING=Off
+
+else
+	cmake -B build -S . \
+		${CMAKE_ARGS} \
+                -G Ninja \
+		-DCMAKE_INSTALL_PREFIX=$PREFIX \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_VERBOSE_MAKEFILE=ON
+fi
+
```

Comment:
Is there a ctest/make check or something that we can run here?

Reply:
The issue is there is one test failing: 
```
      Start 11: nng.platform.resolver_test
11/91 Test #11: nng.platform.resolver_test ..................***Failed    0.03 sec
```